src/main/java/andersen/AndersenTasks/AndersenTasksApplication.java

```diff
@@ -1,43 +1,78 @@
 package andersen.AndersenTasks;
 
 import andersen.AndersenTasks.annotation.NullableWarningProcessor;
-import andersen.AndersenTasks.ticket.Ticket;
+import andersen.AndersenTasks.busticket.BusTicket;
+import andersen.AndersenTasks.busticket.BusTicketType;
+import andersen.AndersenTasks.service.BusTicketService;
 import andersen.AndersenTasks.service.TicketService;
+import andersen.AndersenTasks.ticket.Ticket;
 import andersen.AndersenTasks.user.Admin;
 import andersen.AndersenTasks.user.Client;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+import java.time.LocalDate;
+import java.time.Month;
+import java.util.ArrayList;
+import java.util.Optional;
+
 @SpringBootApplication
 public class AndersenTasksApplication {
 
 	public static void main(String[] args) {
 
-		TicketService ticketService = new TicketService();
-		ticketService.getTicketStorage().forEach(System.out::println);
+		BusTicketService busTicketService = new BusTicketService();
+		busTicketService.busTicketStorage.add(busTicketService.createBusTicketByDateAndType(LocalDate.now(), BusTicketType.DAY));
+		busTicketService.saveBusTicketInStorage(
+				busTicketService.createBusTicketByDateAndType(LocalDate.of(2024, Month.JANUARY,12), BusTicketType.YEAR));
+		busTicketService.busTicketStorage
+				.add(busTicketService.createBusTicketByDateAndType(LocalDate.of(2024,6,18), BusTicketType.MONTH));
+		busTicketService.busTicketStorage
+				.add(new BusTicket(1L, LocalDate.of(2023,12,10), BusTicketType.YEAR, 200));
+		busTicketService.saveBusTicketInStorage(
+				new BusTicket(2L, LocalDate.of(2023,10,1), BusTicketType.YEAR, 6000));
+		busTicketService.saveBusTicketInStorage(
+				new BusTicket( 3L, LocalDate.of(2024,6,17), BusTicketType.WEEK, 1200));
 
-		Ticket ticket = new Ticket();
-		ticket.setProtoId(12L);
+		busTicketService.busTicketStorage.forEach(System.out::println);
 
-		ticket.print();
+		Optional<BusTicket> busTicket = busTicketService.getBusTicketById(1L);
+		System.out.println("busTicket get by id = 1 " + busTicket.get());
 
-		ticket.share("+37068361010");
-		ticket.share("+37068361010","e-kondra@gmail.com");
+		busTicketService.removeBusTicketById(2L);
+		System.out.println("---- remove ticket with id = 2");
+		busTicketService.busTicketStorage.forEach(System.out::println);
 
-		Admin admin = new Admin();
-		admin.printRole();
-		admin.checkTicket(ticket);
+		ArrayList<BusTicket> tickets = busTicketService.getTicketsByTypeAndPrice(BusTicketType.YEAR, 0, 1000);
+		System.out.println("---- getTicketsByTypeAndPrice. TYPE=YEAR, price = from 0 to 1000");
+		tickets.forEach(System.out::println);
 
-		Client client = new Client(ticket);
-		client.printRole();
-		client.getTicket();
 
-		Ticket ticket1 = new Ticket();
-		ticket1.setProtoId(12L);
-		System.out.println(ticket.equals(ticket1));
-		System.out.println(ticket.hashCode());
-		System.out.println(ticket1.hashCode());
+//		TicketService ticketService = new TicketService();
```

Comment:
You should not keep this commented lines in code. The general purpose of VCS is that if you need some code from previous versions of file you could easily go and check  history in git